kubernetes-dev-environment.yaml

```diff
@@ -14,8 +14,8 @@ spec:
     spec:
       restartPolicy: Never
       containers:
-        - name: dev-environment
-          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp:dev-environment
+        - name: allennlp-dev-env
+          image: 896129387501.dkr.ecr.us-west-2.amazonaws.com/allennlp/allennlp-gpu:latest
```

Comment:
It's already there, so I'd rather that not block this PR.  I'm not opposed to a small amount of periphery AI2-specific stuff in the repo, but I'm open to other options as it doesn't make me feel good...